examples/aio-sx/geo-location/geo-location.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: starlingx.windriver.com/v1
+kind: System
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: vbox
+  namespace: geo-location
```

Comment:
Why is this a separate namespace?

Reply:
It was the way I tested it here, but I can reset namespace to deployment and the name to geo-location